spec/reek/source/source_locator_spec.rb

```diff
@@ -9,19 +9,23 @@
       let(:path) { SAMPLES_PATH.join('source_with_hidden_directories') }
 
       let(:expected_paths) do
-        [SAMPLES_PATH.join('source_with_hidden_directories/uncommunicative_parameter_name.rb')]
+        [path.join('uncommunicative_parameter_name.rb')]
       end
 
       let(:paths_that_are_expected_to_be_ignored) do
-        [SAMPLES_PATH.join('source_with_hidden_directories/.hidden/\
-          uncommunicative_parameter_nameicative_method_name.rb')]
+        [path.join('.hidden/uncommunicative_method_name.rb')]
       end
 
       it 'does not scan hidden directories' do
         sources = described_class.new([path]).sources
 
         expect(sources).not_to include(*paths_that_are_expected_to_be_ignored)
-        expect(sources).to eq expected_paths
+      end
+
+      it 'scans directories that are not hidden' do
+        sources = described_class.new([path]).sources
+
```

Comment:
We never did newlines in between 1-line setups and 1-line expectations. Would you like to change this policy?